Lib/test/test_dis.py

```diff
@@ -39,40 +39,40 @@ def cm(cls, x):
 
 dis_c_instance_method = """\
 %3d           0 LOAD_FAST                1 (x)
-              2 LOAD_CONST               1 (1)
+              2 LOAD_CONST               0 (1)
               4 COMPARE_OP               2 (==)
               6 LOAD_FAST                0 (self)
               8 STORE_ATTR               0 (x)
-             10 LOAD_CONST               0 (None)
+             10 LOAD_CONST               1 (None)
              12 RETURN_VALUE
 """ % (_C.__init__.__code__.co_firstlineno + 1,)
 
 dis_c_instance_method_bytes = """\
```

Comment:
This test confuses me every time. The value of the const can't be calculated because co_consts is not provided to dis. So it just prints the index again. But there is no indication that this is the index and not the value. Why is dis doing this? Can we change it? (might as well not print the value, or print `(...)` or something.